backend/src/main/java/com/redhat/cloud/notifications/templates/Ansible.java

```diff
@@ -0,0 +1,64 @@
+package com.redhat.cloud.notifications.templates;
```

Comment:
This file still looks like WIP. Let me know if you need help in here.

This file links the eventTypes to the actual html body and title for each email.

Reply:
@josejulio i will appreciate your help on this, I copied another file and made some changes to that.